src/Void.Core/CommandMode.fs

```diff
@@ -15,23 +16,40 @@ module CommandMode =
     let private requestLanguageForInterpreting buffer =
         buffer, GetCurrentCommandLanguageRequest :> Message
 
-    let handleGetCurrentCommandLanguageResponse buffer response =
+    let private interpretFragment buffer maybeResponse =
         {
-            Language = response.CurrentCommandLanguage
-            Fragment = !buffer
-        } :> Message
+            Language =

```

Comment:
In Haskell, I think this would be an `fmap` and `fromMaybe`. I was surprised to find F# doesn't have a `fromMaybe` equivalent.